__tests__/core/Vector_test.re

```diff
@@ -189,7 +189,7 @@ describe("Vector", () => {
       "should expose a function for linearly interpolating between two integer vectors",
       () =>
       Expect.(
-        expect(Vector.lerp(~acc=v1Int, ~target=v2Int, ~roundness))
+        expect(Vector.lerpV(~acc=v1Int, ~target=v2Int, ~roundness))
```

Comment:
Now that we have `lerp` being used more consistently as a pure mathematical operation on scalars, I figured we should isolate this vector version by appending `V`. I may do this to other vector functions to distinguish them down the road, but not a big deal since we likely won't even ship most of these as part of the public API 👍